jina/parsers/orchestrate/base.py

```diff
@@ -119,3 +119,11 @@ def mixin_base_ppr_parser(parser):
     
     ''',
     )
+
+    gp.add_argument(
```

Comment:
this only applies to executors so it should be in `jina/parsers/orchestrate/runtimes/worker.py`